docs/examples/django/README.rst

```diff
@@ -36,7 +36,7 @@ Execution of the Django app
 
 Set these environment variables first:
 
-#. ``export OPENTELEMETRY_PYTHON_DJANGO_INSTRUMENT=True``
+#. ``export OTEL_DJANGO_INSTRUMENT=True``
```

Comment:
I believe this is supposed to be OTEL_PYTHON_DJANGO_INSTRUMENT, based on the fallback convention of `OTEL_{LANGUAGE}_{FEATURE}`

Reply:
Fixed